<launch>
  <arg name="port" default="/dev/kokam_power" />
  <arg name="rate" default="20" />
  <node name="kokam_power_publisher_node" pkg="vizzy_serial_interfaces" type="kokam_power_publisher" 
  args="$(arg port) $(arg rate)" output="screen"/>
</launch>
